platform/windows/export/export.cpp

```diff
@@ -69,17 +69,17 @@ Error EditorExportPlatformWindows::export_project(const Ref<EditorExportPreset>
 	}
 
 	if (!FileAccess::exists(rcedit_path)) {
-		ERR_PRINTS("Could not find rcedit executable at " + rcedit_path + ", aborting.");
-		return ERR_FILE_NOT_FOUND;
+		ERR_PRINTS("Could not find rcedit executable at " + rcedit_path + ", no icon will be included.");
```

Comment:
> no icon will be included

It's not exclusively for icons, everything in `application/` depends on `rcedit`, app/company name, version, copyright info.

Reply:
So how to call it correctly? Application info? App metadata?

Reply:
>So how to call it correctly? Application info? App metadata?

Microsoft calls it [Version Information](https://docs.microsoft.com/en-us/windows/win32/menurc/versioninfo-resource), I'm not sure what's better.